### ISSUE TITLE
FAQ convert markdown translations to l20n

### ISSUE THREAD

**Issue by pwindle** [Chia-Network/website#132]
I assume you plan on moving from markdown to l20n for the FAQ? I will begin adding the translations into the existing .ftl file per language if that's alright. Let me know if you would like two separate files per language like the markdown currently is.

We could also add URL fragments and perhaps a contents table at the top, so you could link to specific questions. For example to get to the "What is proof of space?" question:
https://chia.net/faq/index.html#What_is_proof_of_space

If you disable javascript on the FAQ page you can see already how this will look. Here's a sample with Deutsch converted:
![image](https://user-images.githubusercontent.com/28284434/37258094-4582a618-256a-11e8-8079-5227fd00502b.png)

**Comment by pwindle** [comment]
#124 which I think is addressing this.
Will hold fire on this for now.

**Comment by dkackman** [comment]
@pwindle https://github.com/Chia-Network/website/pull/124 doesn't change the approach to translating. It still loads them from markdown and transforms them using showdown.js. The only addition is that it supports translation that have both a language and a culture component; i.e. pt-BR or zh-TW.

Changing to your suggested approach would simplify things a bit as there are currently two translation mechanisms. I'm not sure which, if either, is easier to maintain for translators. Probably about the same either way.

**Comment by pwindle** [comment]
Thanks for the info. I think if I can get your locale handling code to work with the l20n js file then it would be functionally equivalent.

Like you say, converting it all to l20n may simplify things a bit, particularly for any future changes to styling and so on, also makes it easier to have all translations in one file which can be split up if you want. I will look into it.

**Comment by ashadle** [comment]
Do you guys know if this had any effect on the styling?  It looks like the old style reverted back on the FAQ page.

**Comment by dkackman** [comment]
@ashadle I think when the styling got fixed it actually broke the translating. And then fixing the translating broke the stying. :(
This PR should fix them both.
https://github.com/Chia-Network/website/pull/124

Another good reason for @pwindle suggestion as the current faq translation implementation is kind fragile and not amendable to much css beyond what I put in the PR I just referenced.

**Comment by pwindle** [comment]
Off topic but any idea why there's no footer on the FAQ now? I can't see why it's broken. It's outside of the markdown container. (Is it just me or has everything become quite a mess recently?)

**Comment by dkackman** [comment]
It looks to me like some git merge conflict markers got commited: https://github.com/Chia-Network/website/blob/2ade1c6be665f031a373f6efd09a65d9060723fd/webroot/faq/index.html#L83

Removing those fixes the footer issue. 

https://github.com/Chia-Network/website/pull/141

**Comment by dkackman** [comment]
@pwindle You made any progress on converting to l20n on the faq page? I'm willing to give it a go if you haven't had a chance.

**Comment by pwindle** [comment]
@dkackman go for it. I had the a lot of it done but there's been a bunch of changes now anyway.

**Comment by dkackman** [comment]
@pwindle do you have a working branch that I could pick up from?

**Comment by pwindle** [comment]
@dkackman I decided to clean it all up. Here : [faq-markdown-to-l20n](https://github.com/pwindle/website/tree/faq-markdown-to-l20n)
Should actually be complete except locale handling code. 

Now I've noticed your neat locale handling code from #124 was merged but isn't in the current master as far as I can tell. What's up with that? EDIT: In my brief testing, locale cookies were all handled correctly without any extra js so I've gone ahead with a pull request,
